build.sbt

```diff
@@ -67,6 +67,29 @@ lazy val coreJS = core.js.settings(
   libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.0.0-RC3" % Test
 )
 
+lazy val epstein = crossProject(JSPlatform, JVMPlatform)
```

Comment:
I think this name has undesirable connotations. How about `zquery` instead?

Reply:
Wow, I did not think of that (had to google it to realize). The reference here was the [Epstein Drive](https://expanse.fandom.com/wiki/Epstein_Drive) from the same series Caliban is from.

Reply:
@desbo That is a great point! It was an attempt to pay homage to @ghostdogpr for inspiring this through his work on Caliban and allude to something that was a part of larger things and made them faster, but clearly this is not worth the negative connotations and the potential to create an unwelcoming environment for some users and contributors. Fixed now. Thanks! 🙏

Reply:
Thank you! It's unfortunate that association exists, appreciate you changing it. The rest of the PR looks awesome btw 😄

Reply:
Thanks!